### PR TITLE
fix: aMap repeat load issure fixed

### PR DESCRIPTION
**Pull request by Cyx649312038**
### This is a ...
- [ ] New feature
- [ ] Improvement
- [x] Bug fix
- [ ] Others

### Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |      aMap repeat load issure fixed     |
| 🇨🇳 Chinese |    高德地图多次调用load方法导致多张地图存在时 部分地图展示报错       |

### Docs

| Language   | Link |
| ---------- | --------- |
| 🇺🇸 English |      |
| 🇨🇳 Chinese |    |

### Checklists
- [x] All changes have been self-tested and work as expected
- [ ] Test cases are updated/provided or not needed
- [ ] Doc is updated/provided or not needed
- [ ] Component demo is updated/provided or not needed
- [ ] Changelog is provided or not needed
- [x] Request a code review if it is necessary

### CODE REVIEW

**Review by katherinehhh — APPROVED** [review]

File: packages/plugins/@nocobase/plugin-calendar/src/locale/zh-CN.ts
Comment (katherinehhh) [inline packages/plugins/@nocobase/plugin-calendar/src/locale/zh-CN.ts:1]: 日历区块的多语言已经处理，这里可以不用处理了，建议去掉

**Review by katherinehhh — COMMENTED** [review]
日历区块的多语言已经处理，这里可以不用处理了，建议去掉

File: packages/plugins/@nocobase/plugin-calendar/src/locale/zh-CN.ts
Comment (Cyx649312038) [reply packages/plugins/@nocobase/plugin-calendar/src/locale/zh-CN.ts:1]: 好的

Comment (Cyx649312038) [reply packages/plugins/@nocobase/plugin-calendar/src/locale/zh-CN.ts:1]: 已删除~

**Pull request merged**